pkg/analyzer/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.39.6
```

Comment:
I *think* this should be 0.39.5?

/cc @stereotype441

Reply:
Yeah, I am not sure what `-dev` means....